worlds/pokemon_emerald/__init__.py

```diff
@@ -288,6 +288,12 @@ def exclude_locations(location_names: List[str]):
                     "Safari Zone SE - Hidden Item in South Grass 2",
                     "Safari Zone SE - Item in Grass",
                 ])
+
+            # Sacred ash is on Navel Rock, which is locked behind the event tickets
+            if not self.options.event_tickets.value:
```

Comment:
You shouldn't need the `.value` here and don't use it in the same constructions elsewhere in this file
```suggestion
            if not self.options.event_tickets:
```

Reply:
I agree with this statement